### DISCUSSION TITLE
Lambda Annotations has no option for VpcConfig

### DISCUSSION THREAD

**Discussion by MrLehiste** [aws/aws-lambda-dotnet#1874] [Q&A] [answered]
I would like to have Lambda VpcConfig in my code, but I can't seem to find an option to do this:
Lambda Annotations overwrites the serverless.template & even if I add VPC in AWS console manually it sometimes gets overwritten.
How can I add persistent VpcConfig information to my Lambdas when using Lambda Annotations?

**Comment by ashishdhingra** [comment]
Need to test this scenario at my end. If we are renaming the function, then essentially, we are deleting the old function and it might cause `VpcConfig` to disappear as well.

**Comment by philasmar** [comment, marked as answer]
In order to add any new resource to the Lambda Annotations project, you will need to manually update the CloudFormation template serverless.template. You mentioned that your changes are being overwritten in serverless.template. I tried updating a sample annotation project with a VPC resource and things worked as expected. Lambda Annotations should only update the CloudFormation resource related to your Lambda functions and it should not affect any other resources you might have added. If you are still having issues, could you please update Lambda Annotations to the latest version? and could you please share the serverless.template with the VPC resource before it gets overwritten?

**Comment by boblodgett** [comment]
@MrLehiste, are you still running into problems on this after looking at philasmar's response?
